### PR TITLE
refactor: get rid of Projects container

### PR DESCRIPTION
**Pull request by thesanjeevsharma**
closes: #2001

### CODE REVIEW

File: src/containers/App.tsx
Comment (thesanjeevsharma) [inline src/containers/App.tsx:40]: lol GitHub Co-pilot 😆

**Comment by jamessspanggg** [comment]
I think we should wait for the designer's decision on the sidemenu before merging this

**Comment by jamessspanggg** [comment]
@thesanjeevsharma i added a temporary fix to add the max-height to the sidemenu, do u think this solution is feasible for now? we can come back to it and fix if the designer has other opinions

**Comment by thesanjeevsharma** [comment]
@jamessspanggg I think we can get rid of this scroll bar?

image

**Review by jamessspanggg — APPROVED** [review]

**Pull request merged**
